Missing repository configuration

The gradle build is missing configuration of at least the alfresco public repository, which leads to a build failure in a clean environment.

```
clean@lars-debian:~/alfresco-zipkin$ ./gradlew build
Downloading https://services.gradle.org/distributions/gradle-5.5-bin.zip
.....................................................................................

Welcome to Gradle 5.5!

Here are the highlights of this release:
 - Kickstart Gradle plugin development with gradle init
 - Distribute organization-wide Gradle properties in custom Gradle distributions
 - Transform dependency artifacts on resolution

For more details see https://docs.gradle.org/5.5/release-notes.html

Starting a Gradle Daemon (subsequent builds will be faster)

> Configure project :docker:repo
using default credentials

FAILURE: Build failed with an exception.

* What went wrong:
A problem occurred configuring project ':docker:repo'.
> Could not resolve all files for configuration ':docker:repo:baseAlfrescoWar'.
   > Could not find org.alfresco:alfresco-platform:5.2.g.
     Searched in the following locations:
       - https://plugins.gradle.org/m2/org/alfresco/alfresco-platform/5.2.g/alfresco-platform-5.2.g.pom
     Required by:
         project :docker:repo

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

Deprecated Gradle features were used in this build, making it incompatible with Gradle 6.0.
Use '--warning-mode all' to show the individual deprecation warnings.
See https://docs.gradle.org/5.5/userguide/command_line_interface.html#sec:command_line_warnings

BUILD FAILED in 25s
```

--- a/build.gradle
+++ b/build.gradle
@@ -2,6 +2,14 @@ allprojects {
     apply plugin: 'java'
     apply plugin: 'idea'
 
+    repositories {
+        mavenCentral()
+        jcenter()
+        maven {
+            url 'https://artifacts.alfresco.com/nexus/content/repositories/public/'
+        }
+    }
+
     sourceCompatibility = 1.8
     targetCompatibility = 1.8
 
@@ -17,7 +25,7 @@ allprojects {
         version = "$baseVersion-SNAPSHOT"
 
     project.ext {
-        alfresco_version = '5.2.5'
+        alfresco_version = '5.2.f'
         brave_version = '5.6.9'
         zipkin_reporter_version = '2.10.0'
     }

